AnyType/Project_2/AnyType.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+#include<iostream>
+
+class AnyType
+{
+private:
+	Types ValueType;//current type of the data
+	Data data;//current data 
+
+public:
+	class BadCastException : public std::exception
```

Comment:
Declaration of exception inside class AnyType. It would be better if you declare your own exception outside AnyType.